ansible/local_testing/Vagrantfile

```diff
@@ -2,19 +2,20 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "bento/debian-11"
-    config.vm.box_version = "202212.11.0"
+    config.vm.box = "bento/debian-12.5"
```

Comment:
Should we just pin to `bento/debian-12` to automatically get new patches? They shouldn't break anything, it's Debian after all

Reply:
Yeah that sounds good, I'll make the change